example/components/PlaidLink.tsx

```diff
@@ -1,39 +1,41 @@
 import React from 'react';
 import {
     Text,
-    TouchableOpacity,
+    View,
     StyleSheet,
 } from 'react-native';
-import {PlaidLink, usePlaidEmitter} from 'react-native-plaid-link-sdk/PlaidLink';
-import {useNavigation} from '@react-navigation/native';
-import {LinkExit} from "react-native-plaid-link-sdk/types/Types";
-import {LinkSuccess} from "react-native-plaid-link-sdk/types/Types";
+import { PlaidLink, usePlaidEmitter } from 'react-native-plaid-link-sdk/PlaidLink';
+import { useNavigation } from '@react-navigation/native';
```

Comment:
What's our policy on imports?

When importing multiple types from the same file, do we:
- use a separate line for each import
- combine all imports into a single import statement

Now we're doing both.

Reply:
good point, prettier only breaks to a new line when the import statement is long but doesn't break if there is a single import https://prettier.io/docs/en/rationale.html#imports

We should be combining all imports from the same package though